app/controllers/players/index.js

```diff
@@ -1,5 +1,27 @@
 import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 
 export default class IndexController extends Controller {
+    queryParams = [
+        'sortField',
+        'sortDirection'
+    ];
 
+    @tracked sortField = 'id';
+    @tracked sortDirection = 'ASC';
+
+    @action
+    onHeaderClick(fieldName) {
+        this.sortField = fieldName;
+
+        switch (this.sortDirection) {
```

Comment:
We can use a simple `if/else` condition or ternary operator here as there are only two possible options. What do you think about it?